explorer/app/components/Section/section.stories.tsx

```diff
@@ -1,52 +1,35 @@
 // Core dependencies
```

Comment:
It looks like I am still seeing the old version of this story:

<img width="1252" alt="image" src="https://user-images.githubusercontent.com/2446424/175090843-5788df28-6f73-4052-8b1a-9a6ce5a382a3.png">


Any thoughts?